app/experimenter/static/rapid/components/forms/ExperimentForm.tsx

```diff
@@ -37,6 +37,37 @@ const ExperimentForm: React.FC = () => {
   );
   const history = useHistory();
 
+  const featureOptions = [
+    { value: "FEATURE 1", label: "FEATURE 1" },
+    { value: "FEATURE 2", label: "FEATURE 2" },
+  ];
+
+  const audienceOptions = [
+    { value: "AUDIENCE 1", label: "AUDIENCE 1" },
+    { value: "AUDIENCE 2", label: "AUDIENCE 2" },
+  ];
+
+  const handlefeatureChange = (featureValues) => {
```

Comment:
nit: should this be `handleFeatureChange()` (upper "F") to be consistent with `handleAudienceChange()` below?